Owl/app/src/main/java/com/example/owl/ui/NavGraph.kt

```diff
@@ -16,37 +16,26 @@
 
 package com.example.owl.ui
 
-import android.os.Parcelable
-import androidx.compose.runtime.Immutable
-import com.example.owl.ui.utils.Navigator
-import kotlinx.android.parcel.Parcelize
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.navigate
 
-/**
- * Models the screens in the app and any arguments they require.
- */
-sealed class Destination : Parcelable {
-    @Parcelize
-    object Onboarding : Destination()
-
-    @Parcelize
-    object Courses : Destination()
-
-    @Immutable
-    @Parcelize
-    data class Course(val courseId: Long) : Destination()
+object Destinations {
+    const val Onboarding = "Onboarding_dest_key"
```

Comment:
I would suggest using simply `"Onboarding"` as a value here. That way, deeplinks will be more readable and sensible.